pkgs/development/libraries/kde-frameworks/kde-wrapper.nix

```diff
@@ -61,13 +61,9 @@ stdenv.mkDerivation {
         fi
     done
 
-    ln -s "$env/share" "$out"
```

Comment:
You're definitely right to remove this. What I can't decide is: do we need to instead link the `/share` paths from the unwrapped packages? I think probably not, for standalone applications, so don't worry about that yet.